AppDashboard/lib/app_dashboard_data.py

```diff
@@ -170,6 +172,75 @@ def __init__(self, helper=None):
     """
     self.helper = helper or AppDashboardHelper()
 
+  def build_dict(self):
+    LOOKUP_DICT = {}
+    LOOKUP_DICT['cloud_stats'] = {"title":"Cloud Status",
+                                  "link":"/status/cloud",
+                                  "template":"status/cloud.html"}
+    LOOKUP_DICT['database_stats'] = {"title":"Database",
+                                  "template":"apps/database.html"}
+    LOOKUP_DICT['memcache_stats'] = {"title":"Memcache",
+                                     "template":"apps/memcache.html"}
+    LOOKUP_DICT['upload_app'] = {"title":"Upload App",
+                                 "link":"/apps/new",
+                                 "template":"apps/new.html"}
+    LOOKUP_DICT['delete_app'] = {"title":"Delete App",
+                                 "link":"/apps/delete",
+                                 "template":"apps/delete.html"}
+    LOOKUP_DICT['manage_users'] = {"title":"Manage Users",
+                                   "link":"/authorize",
+                                   "template":"authorize/cloud.html"}
+    LOOKUP_DICT['logging'] = {"title":"Logs",
+                              "link":"/logs",
+                              "template":"logs/main.html"}
+    LOOKUP_DICT['monit'] = {"title":"Monit",
+                             "link":self.get_monit_url()}
+    LOOKUP_DICT['taskqueue'] = {"title":"TaskQueue",
+                             "link":self.get_flower_url()}
+
+    LOOKUP_DICT['app_management'] = [self.helper.can_upload_apps(),
+                                     {"App Management":
+                                        [{"upload_app": LOOKUP_DICT[
+                                          'upload_app']},
+                                         {"delete_app": LOOKUP_DICT[
+                                           'delete_app']}]}]
+    # TODO: remove, used temporarily so user does not have to log in
+    LOOKUP_DICT['app_management'] = ["True",
```

Comment:
When dealing with boolean values, use Python's `True` and `False`. You can then test it directly with `if value` rather than a string equality like `if value == 'True'`.